### DISCUSSION TITLE
Staatliche Hochschule für Musik, Trossingen

### DISCUSSION THREAD

**Discussion by opensourceguitars** [leihs/leihs#1269] [Show and tell]
Hallo,

wir benutzen Leihs seit ca. 1.5 Jahren für unseren Geräte-Leihpool https://www.mh-trossingen.de/infothek/fuer-studierende/leihpool.html

Vielleicht könnt ihr uns ja im Git-Readme erwähnen, unsere Hochschulwebseite ist derzeit:
 http://mh-trossingen.de

Viele Grüße,

Joachim

**Comment by DrTom** [comment]
Here you go https://github.com/leihs/leihs/commit/0ed88ee3761e697769d96e99ed902f5280626fd2

thanks a lot
